packages/fuelet_secure_layer/lib/src/features/account/entity/account.dart

```diff
@@ -45,8 +45,13 @@ class Account with EquatableMixin {
     this.walletGroup = WalletGroup.myWallets,
     this.addingMethod = AccountAddingMethod.notDetected,
     this.derivativeInfo,
-    this.name,
-  });
+    String? name,
+  }) : name = name != null
+            ? name.replaceAllMapped(
+                RegExp(r'Account|account'),
+                (match) => match[0] == 'Account' ? 'Address' : 'address',
```

Comment:
Let's replace just `Account` and ignore `account`. By default we create account names with the uppercase letter. If there's `account` in the name, that means that the user named it himself, so we should keep that name